src/domains/ecommerce-app/.terraform.lock.hcl

```diff
@@ -6,6 +6,7 @@ provider "registry.terraform.io/hashicorp/azuread" {
   constraints = "2.21.0"
   hashes = [
     "h1:9gG6SWoUZZmmXbYBv6ra2RF5NYpamB9tGjsuBxrasFQ=",
+    "h1:KbY8dRdbfTwTzEBcdOFdD50JX8CUG5Mni25D2+k1rGc=",
```

Comment:
👀

Reply:
to revert

Reply:
Ops, missed it. Fixed with https://github.com/pagopa/pagopa-infra/pull/794/commits/e2f112cc04474682ee2bcae14676ef06a194ad6e